proto/dwtools/abase/l0.test/Long.test.s

```diff
@@ -13511,16 +13511,12 @@ function longFromRangeWithStep( test )
   test.case = 'wrong arguments';
   test.shouldThrowErrorSync( () => _.longFromRangeWithStep( 'wrong arguments' ) );
   test.shouldThrowErrorSync( () => _.longFromRangeWithStep( undefined ) );
+  test.shouldThrowErrorSync( () => _.longFromRangeWithStep( [ 0, 1 ] ) );
+  test.shouldThrowErrorSync( () => _.longFromRangeWithStep( [ 0, 1 ], 1, 2 ) );
```

Comment:
The test case 
```js
  test.case = 'more then 2 arguments'
  test.shouldThrowErrorSync( () => _.longFromRangeWithStep( [ 0, 1 ], 1, 2 ) );
``` 
is good case. 
This is a separate test case. Move it back, please.